File: LOOPS/FIZZBUZZ.py
# Lizzie DeLong, FizzBuzz in Python

for num in range(1,51):
    words = []
    if num % 3 == 0:
        words.append("Fizz")
    if num % 5 == 0:
        words.append("Buzz")
    if not words:
        print(num)
    else:
        print(''.join(words))

File: LOOPS/FIZZBUZZ.c
// Lizzie DeLong, FizzBuzz in C
#include <stdio.h>

int main(void){
    int i;
    for(i = 1; i <= 50; i++){
        if(i % 3 == 0 && i % 5 ==0){
            printf("FizzBuzz\n");
        } else if(i % 5 == 0) {
            printf("Buzz\n");
        } else if(i % 3 == 0){
            printf("Fizz\n");
        } else {
            printf("%d\n", i);
        }
    }
    return 0;
}